custom/icds_reports/reports/issnip_monthly_register.py

```diff
@@ -94,32 +95,32 @@ def css_record_monthly(self):
                 }, 'lactating')
             )
         )
-        return data[0] if data else None
+        return data
 
     @cached_property
     def infrastructure_data(self):
-        data = AWCInfrastructureUCR(self.config).data
-        return list(data.values())[-1] if data else None
+        data = AWCInfrastructureUCR(self.config.copy()).data
```

Comment:
Why did you add the `.copy()` here? can these classes modify the config?

Reply:
@emord Because ```clean_IN_filter_value(self.config, 'awc_id')``` function modify the config.

Reply:
Couldn't you do that once and pass in the cleaned version?